.github/workflows/CI.yml

```diff
@@ -186,3 +186,18 @@ jobs:
     - name: Format
       run: |
         black --check .
+
+  check-codeblock-coverage:
+    runs-on: ubuntu-latest
```

Comment:
You should pin the version

Reply:
I might be missing something but none of the other jobs specify an ubuntu version?